### DISCUSSION TITLE
posts/%E6%A8%A1%E6%9D%BF%E6%B5%8B%E8%AF%95/

### DISCUSSION THREAD

**Discussion by giscus** [TonyMarsh31/TonyMarsh31.github.io#1] [Announcements]
# posts/%E6%A8%A1%E6%9D%BF%E6%B5%8B%E8%AF%95/

This is a test post.

https://tonymarsh31.github.io/posts/%E6%A8%A1%E6%9D%BF%E6%B5%8B%E8%AF%95/

**Comment by TonyMarsh31** [comment]
this is a comment test
